docs/literate/src/files/first_steps/create_first_setup.jl

```diff
@@ -19,7 +19,9 @@
 
 # The first step is to create and open a file with the .jl extension. You can do this with your
 # favorite text editor (if you do not have one, we recommend [VS Code](https://code.visualstudio.com/)).
-# In this file you will create your setup.
+# In this file you will create your setup. Alternatively, you can execute each line of the
+# following code one by one in the Julia REPL. This will generate useful output for nearly every
```

Comment:
```suggestion
# In this file you will create your setup and the file can then be executed in Julia using, for example, `trixi_include()`.
# Alternatively, you can execute each line of the following code one by one in the 
# Julia REPL. This will generate useful output for nearly every
```
Not sure if you would refer to `trixi_include()` or `include()` ...